benchexec/tablegenerator/react-table/src/components/ReactTable.js

```diff
@@ -665,37 +674,40 @@ const Table = (props) => {
 
   const renderHeaderGroup = (headerGroup) => (
     <div className="tr headergroup" {...headerGroup.getHeaderGroupProps()}>
-      {headerGroup.headers.map((header) => (
-        <div
-          {...header.getHeaderProps({
-            className: `th header ${header.headers ? "outer " : ""}${
-              header.className
-            }`,
-          })}
-        >
+      {headerGroup.headers.map((header) => {
```

Comment:
Is there any relevant change in this part of the code, or is it just reformatting? Looks like the latter to me. In this case, please keep only the relevant fix, not the reformatting.